src/cache-restore.ts

```diff
@@ -22,9 +25,17 @@ export const restoreCache = async (packageManager: string) => {
     packageManagerInfo,
     packageManager
   );
-  const lockFilePath = findLockFile(packageManagerInfo);
+  const lockFilePath = cacheDependencyPath
+    ? cacheDependencyPath
```

Comment:
Question.

It looks like the raw `cacheDependencyPath` path input is used as the input to `glob.hashFiles`.

Lets imagine I have 2 different inputs that lead to the same dependency files.

In the first I have
```
    cache-dependency-path: |
      server/app/package-lock.json
      frontend/app/package-lock.json
```

While in the second i use
```
    cache-dependency-path: **/*package-lock.json
```

Would the `fileHash` generated by both inputs be the same?

Reply:
If the repository only contains these two `package-lock.json` files, the `fileHash` will be the same for both cases.